### ISSUE TITLE
Add ability to edit the stack values

### ISSUE THREAD

**Issue by tur11ng** [Vector35/debugger#197]

**Comment by xusheng6** [comment]
Currently, we do support editing the stack values (as well as any other values) in the linear view or hex view. Just switch to the hex view, navigate to the stack pointer address and edit it directly. We will also a more convenient way to edit data variable values in the linear view as well. 

We may or may not support viewing/editing stack variable values in the stack value widget in the status bar, as we feel its more straightforward to do in linear/hex view. We have not ruled out support for it though.

**Comment by tur11ng** [comment]
Yes they are related, I posted it here also because it is related to the IL API too.

**Comment by xusheng6** [comment]
Duplicate of https://github.com/Vector35/debugger/issues/322.
